app/views/editor/_toolbar.html.erb

```diff
@@ -14,34 +14,34 @@
   </div>
   <!-- defined in application_helper.rb -->
   <% toolbar_element_id = get_toolbar_element_id(location, local_assigns[:reply_to], local_assigns[:comment_id]) %>
-  <div class="btn-group mr-2 ">
+  <div class="btn-group mr-2">
     <a
       id="bold-button-<%= toolbar_element_id %>"
-      data-toggle="tooltip" 
       title="Bold" 
+      data-action="bold"
       data-placement="bottom" 
-      class="bold-button btn btn-outline-secondary btn-sm" 
-      onClick="$E.bold()"
+      data-toggle="tooltip" 
+      class="bold-button btn btn-outline-secondary btn-sm rich-text-button" 
     >
```

Comment:
Again, note that I removed the onclick function. I also added a `data-action` attribute which is used in the button click handler.